examples/gnn_fraud_detection_pipeline/stages/graph_sage_stage.py

```diff
@@ -55,21 +56,15 @@ def __init__(self,
 class GraphSAGEStage(SinglePortStage):
 
     def __init__(self,
-                 c: Config,
-                 model_hinsage_file: str,
-                 batch_size: int = 5,
-                 sample_size: typing.List[int] = [2, 32],
+                 config: Config,
+                 model_dir: str,
+                 batch_size: int = 100,
                  record_id: str = "index",
                  target_node: str = "transaction"):
-        super().__init__(c)
+        super().__init__(config)
 
-        # Must import stellargraph before loading the model
-        import stellargraph.mapper  # noqa
-        import tensorflow as tf
-
-        self._keras_model = tf.keras.models.load_model(model_hinsage_file)
+        self._dgl_model, _, self.hyperparam = load_model(model_dir)
```

Comment:
`self.hyperparam` doesn't appear to be in use. Maybe instead:
```suggestion
        self._dgl_model, _, __ = load_model(model_dir)
```